src/Console/ConfigurationResolver.php

```diff
@@ -401,52 +403,69 @@ private function resolveConfigPath()
             return;
         }
 
-        if (null === $this->path) {
+        if (empty($this->path)) {
```

Comment:
This is no longer possible I think?
0 length can only be with `stdIn` which is covered at line 400?